backend/internal/api/litellm_test.go

```
@@ -0,0 +1,365 @@
+}
+
+func TestLiteLLMUnsupportedMethodOnCollection(t *testing.T) {
+	fixture := newLiteLLMFixture(t)
+	if got := liteLLMRequest(t, fixture, http.MethodPatch, "/api/v1/litellm", nil, true).Code; got != http.StatusNotFound {
+		t.Fatalf("patch collection status=%d", got)
+	}
+}
+
+func TestLiteLLMStatusInternalError(t *testing.T) {
+	fixture := newLiteLLMFixture(t)
+	if _, err := fixture.db.Exec("DROP TABLE manager_settings"); err != nil {
```

golangci-lint (2.13.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)